### PR TITLE
 [8.0][MIG][website_form_recaptcha] Backport from v9. 

### PR DESCRIPTION
**Pull request by yajo**
Remove references to `website_form` module, not required to work and not available in v8.

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can this be a snippet that can be dragged?

File: website_form_recaptcha/README.rst [hunk 1]
```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Website Form - ReCaptcha
+========================
+
+Adds a ReCaptcha field widget for website forms
```
Comment (pedrobaeza) [inline website_form_recaptcha/README.rst:9]: Is widget the best word here?

File: website_form_recaptcha/i18n/en.po [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Translation of Odoo Server.
```
Comment (pedrobaeza) [inline website_form_recaptcha/i18n/en.po]: Don't include this file.

File: website_form_recaptcha/i18n/pt_BR.po [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Language: pt_BR\n"
+"Plural-Forms: nplurals=2; plural=(n > 1);\n"
+
+#. module: website_form_recaptcha
+#: model:ir.model.fields,field_description:website_form_recaptcha.field_website_form_recaptcha_display_name
+msgid "Display Name"
+msgstr "Mostrar Nome"
```
Comment (pedrobaeza) [inline website_form_recaptcha/i18n/pt_BR.po]: Nombre mostrado

**Comment by pedrobaeza** [comment]
Include Spanish translation

File: website_form_recaptcha/README.rst (hunk 1 above)
Comment (yajo) [inline website_form_recaptcha/README.rst:9]: well, it's a widget Google's POV, but maybe not Odoo's.

**Comment by yajo** [comment]
Making a draggable snippet makes more sense in v9, because enterprise has the form builder. In v8 it could appear, but honestly I don't see it worth the time, at least for now.

**Comment by lasley** [comment]
LGTM 👍

**Comment by pedrobaeza** [comment]
Travis is failing.

**Comment by lasley** [comment]
Ah yes good point, I should have probably looked there too!

**Comment by rafaelbn** [comment]
Please @Yajo review travis, thanks!

**Comment by yajo** [comment]
Travis should work now, but now that I have used it, I have to admit it... [`mock` rocks!](https://docs.python.org/3/library/unittest.mock.html) :guitar:

**Comment by rafaelbn** [comment]
:+1:

**Comment by lasley** [comment]
Mock is the single most magical library in existence! Makes you feel like some sort of code surgeon or something 😛 

LGTM 👍

**Pull request merged**